cmd/upgrade.go

```diff
@@ -32,7 +32,7 @@ We were not able to upgrade the cli because we encountered an error:
 
 Please check the FAQ for solutions to common upgrading issues.
 
-https://exercism.io/faqs`, err)
+https://exercism.org/faqs`, err)
```

Comment:
As per the forum post, this should be:

```suggestion
https://exercism.org/docs/using/faqs`, err)
```

Reply:
I have adjusted this according to comment :) 
https://github.com/exercism/cli/pull/1124#pullrequestreview-1804909560

Reply:
@tomasnorre I don't see the updated version?

Reply:
See closed resolved conversation between @iHiD and me.

![github-exercism](https://github.com/exercism/cli/assets/1212481/9d491c3f-59db-4d88-8b8c-5aa5a0a4afec)

That's why I haven't changed it back to your suggestion.

Reply:
Oh! I didn't see that because it was resolved. Makes a ton of sense then, sorry!

Reply:
Resolved in https://github.com/exercism/website/pull/6624